streaming/index.js

```diff
@@ -441,6 +444,7 @@ const startServer = async () => {
         return;
       }
 
+      // @ts-ignore
```

Comment:
This is because the types for `pg` don't declare it as a union on `Pool.connect` of `(err: Error, client: never, done: (release?: any) => void) | (err: never, client: PoolClient, done: (release?: any) => void)` which would allow discrimination of the types, so instead you get an error of "client may be undefined" for `client.query`

We could alternatively use async/await here, which would side-step that issue of `Pool.connect`'s types being incorrect since: `const client = await pgPool.connect()` is correctly typed as `Promise<PoolClient>;`